projects/openapi-utilities/src/openapi3/traverser.ts

```diff
@@ -63,10 +145,16 @@ export class OpenApiV3Traverser implements Traverse<OpenAPIV3.Document> {
 
     for (let [pathPattern, paths] of Object.entries(this.input.paths || {})) {
       const traverser = this;
+      yield {
+        location: {
+          jsonPath: jsonPointer.append('', 'paths', pathPattern),
+        },
+        type: 'path',
+      };
```

Comment:
@acunniffe I needed to add this because
- the diff will dig down to the lowest level where there's a diff between two objects
- The case where we add a path `/paths/~1api~1users` is not a diff at an operation level, it's the level above. The way we group diffs into relevant nodes is by looking at the nearest parent fact, which in this case is a specification. This wasn't a problem with previous diffs because we were comparing facts to facts, instead of where the diffs occured